packages/mobile/src/identity/verification.ts

```diff
@@ -319,6 +330,8 @@ export function* doVerificationFlow(withoutRevealing: boolean = false) {
       }
 
       yield put(setVerificationStatus(VerificationStatus.CompletingAttestations))
+      // NOTE: I think we want the DEK registration out of the race because
```

Comment:
I definitely agree with this. Related to the edge case mentioned in #5622, wallet address registration should be a requirement before the attestation is complete. My recommendation would be to move this such that it occurs in parallel with the verification flow, and check here to make sure it is complete before finalizing the attestation,

Reply:
I've moved it out of the race condition but don't believe any further changes are necessary for now. I think the edge case in #5622 pertains only to feeless verification. If DEK/wallet registration fails for non-feeless verification, then our new logic will correctly assume that `accountAddress === walletAddress`.